src/error.ts

```
@@ -89,9 +97,12 @@ export function errorSnapshot(
 ): string {
     const message = [header];
     if (err instanceof BaseError) {
+        const org = getRpcError(err);
         if (err.shortMessage) message.push("Reason: " + err.shortMessage);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)